modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/package.json

```diff
@@ -2,6 +2,7 @@
 	"dependencies": {
 		"@clayui/button": "3.0.3",
 		"@clayui/form": "3.2.1",
+		"@clayui/modal": "3.1.3",
```

Comment:
Hey @oliveiraaraujo, this is likely to cause SF errors because you're not using the same version as the rest of `master` which I think is already `3.1.4`.

See the changes in `yarn.lock` below